### PR TITLE
{vis}[GCC/11.3.0,GCCcore/11.3.0] GSL v2.7, Wayland v1.20.0, PyCairo v1.21.0, ... w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by jfgrimm**
depends on:

- [x] #15445 

(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GST-plugins-bad/GST-plugins-bad-1.19.2-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+easyblock = 'MesonNinja'
+
+name = 'GST-plugins-bad'
+version = '1.19.2'
```
Comment (branfosj) [inline easybuild/easyconfigs/g/GST-plugins-bad/GST-plugins-bad-1.19.2-GCC-11.3.0.eb:4]: There is a 1.20.2 for this as well: https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.20.2.tar.xz

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/GST-plugins-bad/GST-plugins-bad-1.19.2-GCC-11.3.0.eb (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/g/GST-plugins-bad/GST-plugins-bad-1.19.2-GCC-11.3.0.eb:4]: hmm, somehow overlooked that

Comment (jfgrimm) [reply easybuild/easyconfigs/g/GST-plugins-bad/GST-plugins-bad-1.19.2-GCC-11.3.0.eb]: @branfosj bumped to 1.20.2

File: easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('Meson', '0.62.1'),
+    ('Ninja', '1.10.2'),
+    ('pkgconf', '1.8.0'),
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.3.0.eb:24]:
```suggestion
    ('binutils', '2.38'),
]
```

Add `binutils` to fix the pipeline

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 9 out of 9 (8 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/e7d0573e6bb3e84b96efaf52bbbd4bb4 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/79f9d98f581f089842670a39540ece1e for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 9 out of 9 (8 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/cdd8d141096588f9a1552a41d889a491 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 9 out of 9 (8 easyconfigs in total)
bear-pg0306u11a.bear.cluster - Linux RHEL 8.5, POWER, 8335-GTX (power9le), 4 x NVIDIA Tesla V100-SXM2-16GB, 470.57.02, Python 3.6.8
See https://gist.github.com/50079a98c84c8e8bb58ad1182eb04a64 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**